news/blog/_posts/2021-04-19-cadence-java-3-0-0.md

```diff
@@ -1,5 +1,5 @@
 ---
-date: 2021-04-19T07:00:02
+date: 2021-04-19T07:00:01
```

Comment:
?

Reply:
controls ordering of posts. Latest datetime stamp will appear first in the list of news items.